common/src/main/java/io/honeycomb/opentelemetry/EnvironmentConfiguration.java

```diff
@@ -129,7 +129,7 @@ public static int getSampleRate() throws NumberFormatException {
      * Get a friendly error message for missing variable.
      *
      * @param humanKey human-friendly variable description
-     * @param key environment variable key
+     * @param key      environment variable key
```

Comment:
weird space got added?
```suggestion
     * @param key environment variable key
```

Reply:
Ah, the opposite -- previously it wasn't formatted properly